src/app/+item-page/edit-item-page/item-delete/item-delete.component.ts

```diff
@@ -1,31 +1,285 @@
-import { Component } from '@angular/core';
-import { first } from 'rxjs/operators';
-import { AbstractSimpleItemActionComponent } from '../simple-item-action/abstract-simple-item-action.component';
-import { getItemEditPath } from '../../item-page-routing.module';
-import { RestResponse } from '../../../core/cache/response.models';
+import {Component, Input, OnInit} from '@angular/core';
+import {filter, first, map, switchMap, take} from 'rxjs/operators';
+import {AbstractSimpleItemActionComponent} from '../simple-item-action/abstract-simple-item-action.component';
+import {getItemEditPath} from '../../item-page-routing.module';
+import {NgbModalRef} from '@ng-bootstrap/ng-bootstrap';
+import {combineLatest as observableCombineLatest, combineLatest, Observable} from 'rxjs';
+import {RelationshipType} from '../../../core/shared/item-relationships/relationship-type.model';
+import {VirtualMetadata} from '../virtual-metadata/virtual-metadata.component';
+import {Relationship} from '../../../core/shared/item-relationships/relationship.model';
+import {getRemoteDataPayload, getSucceededRemoteData} from '../../../core/shared/operators';
+import {hasValue, isNotEmpty} from '../../../shared/empty.util';
+import {Item} from '../../../core/shared/item.model';
+import {MetadataValue} from '../../../core/shared/metadata.models';
+import {ViewMode} from '../../../core/shared/view-mode.model';
```

Comment:
Again, some unnecessary import reformatting.

Reply:
I've reverted the imports reformatting